openvdb/openvdb/tree/Tree.h

```diff
@@ -350,15 +350,15 @@ class Tree: public TreeBase
     /// Return the number of non-leaf nodes.
     Index32 nonLeafCount() const override { return mRoot.nonLeafCount(); }
     /// Return the number of active voxels stored in leaf nodes.
-    Index64 activeLeafVoxelCount() const override { return mRoot.onLeafVoxelCount(); }
+    Index64 activeLeafVoxelCount() const override { return tools::countActiveLeafVoxels(*this); }
     /// Return the number of inactive voxels stored in leaf nodes.
-    Index64 inactiveLeafVoxelCount() const override { return mRoot.offLeafVoxelCount(); }
+    Index64 inactiveLeafVoxelCount() const override { return tools::countInactiveLeafVoxels(*this); }
     /// Return the total number of active voxels.
     Index64 activeVoxelCount() const override { return tools::countActiveVoxels(*this); }
     /// Return the number of inactive voxels within the bounding box of all active voxels.
-    Index64 inactiveVoxelCount() const override;
+    Index64 inactiveVoxelCount() const override { return tools::countInactiveVoxels(*this); }
     /// Return the total number of active tiles.
-    Index64 activeTileCount() const override { return mRoot.onTileCount(); }
+    Index64 activeTileCount() const override { return tools::countActiveTiles(*this); }
```

Comment:
As these are virtual, do we ever see them being deprecating?

Reply:
I don't see this being much different to `Tree::activeVoxelCount()` which we agreed would be too disruptive to deprecate at this stage. Perhaps some of these other methods are less popular (`Tree::inactiveLeafVoxelCount()`?), but if I don't really see the point in removing some but not others.

What is more practical perhaps is to deprecate some of the intermediate calls such as `RootNode::onTileCount()`.